src/commands/generateBugReport.ts

```diff
@@ -169,50 +169,50 @@ Outdated:\t${info.outdated}
 
   async getRuntimeInfo(): Promise<TerraformInfo> {
     const terraformExe = 'terraform';
-
     const spawn = child_process.spawnSync;
 
-    try {
-      const child = spawn(terraformExe, ['version', '-json']);
-      const response = child.stdout.toString();
-      const j = JSON.parse(response);
-
-      return {
-        version: j.terraform_version,
-        platform: j.platform,
-        outdated: j.terraform_outdated,
-      };
-    } catch (err) {
-      if (err.status === 0) {
+    // try to get version from a newer terraform binary
+    const resultJson = spawn(terraformExe, ['version', '-json']);
```

Comment:
This may be out of scope of this PR because it's a refactoring and not strictly related to Typescript strict checking, but since we have to change the set of code that executes a binary in two places, I think it would make sense to extract to a single function that we can reference in both places.

Reply:
Thanks for your review!

I'm not sure which part of the code you're suggesting to extract here. Can you explain?

Reply:
I'm referring to the parts where we detect the version of terraform and the version of terraform-ls. Different places in the extension, but nearly the same code to be executed to try/catch around the version parameter being present and/or json being accepted.

Reply:
You're right. It would make sense to consolidate this into a single helper function. However, since this refactoring might also affect parts of the language server installer, I suggest we do it in an extra PR after everything has been merged.